src/components/formula.jsx

```diff
@@ -26,8 +26,8 @@ class Formula extends React.Component {
           <h3>
             <FormattedMessage id={stopDate ? 'fromToDate' : 'fromDate'}
               values={{
-                startDate: startDate && <FormattedDate value={startDate} year="numeric" month="2-digit" day="2-digit" />,
-                stopDate: stopDate && <FormattedDate value={stopDate} year="numeric" month="2-digit" day="2-digit" />,
+                startDate: startDate && <FormattedDate value={startDate} year="numeric" month="2-digit" day="2-digit" timeZone="UTC"/>,
```

Comment:
> As no timezone is specified in `startDate`, ReactIntl assumes it's a UTC date. However, it will try to display it in the local timezone, hence the offset.
> Forcing the date formatting to UTC prevents this offset

Reply:
We can't give dates in UTC. We'd have to ask the user if it happened before or after 1pm, which isn't very good UX. Generally the conditions like a parameter change, or a birthday is reached at midnight in the timezone, not midnight in UTC

Reply:
Maybe the timezone is a configuration variable. We can configure that dates and times provided by the user are in NZ timezones. Most people won't understand the concept of providing for example their date-of-birth in UTC.